### ISSUE TITLE
Bug: --only-licenses isn't working

### ISSUE THREAD

**Issue by jochman** [FHPythonUtils/LicenseCheck#80]
- [x] I have checked for similar issues in the project's issue tracker.
- [x] I have searched closed issues to see if a similar problem was reported
  before.

### Description

The new feature `--only-licenses` isn't working from pr https://github.com/FHPythonUtils/LicenseCheck/pull/71

### Expected Behavior

To have this option.

### Actual Behavior

```sh
licensecheck --only-licenses mit
licensecheck: error: unrecognized arguments: --only-licenses
```

## System Information

Please provide the following additional information about your system or
environment:

- Operating System (OS): MacOS
- OS Version: 
- Project Version (if applicable): 2.2024 & master branch (commit 979ded34a87fa6525c2b1dda2d0d29bd166cbdc9)

**Comment by FredHappyface** [comment]
I can confirm this exists on version 2024.2 (https://github.com/FHPythonUtils/LicenseCheck/tree/2024.2), and on the main brainch 

Cannot reproduce locally 

![image](https://github.com/user-attachments/assets/9d3d176d-85ab-47bb-8e10-f3e7ec0f77b4)
